app/views/batches/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1><span class="fa fa-copy"></span> Batch <%= @batch.job_id %></h1>
+
+<div class='table-responsive'>
+  <table class='table table-striped'>
+    <thead>
+      <tr>
+        <th scope='col'>Status</th>
+        <th scope='col'>Row Number</th>
+        <th scope='col'>Accession Number</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @batch.batch_items.each do |item| %>
+        <tr>
+          <td><%= status_label item.status %></td>
+          <td><%= item.row_number %></td>
+          <td><%= item.accession_number %></td>
```

Comment:
do you think you just spit out the error if something goes wrong? Something like this?

```
          <td>
            <% item.error.each do |error|  %>
              <%= error %>
            <% end %>
          </td>
```

Since I'm not seeing it in the log either, I think that will make things alot clearer right now. 

After that I'm cool with merging.